application_form/api/views.py

```diff
@@ -8,9 +9,11 @@ class HasoApplicationViewSet(viewsets.ModelViewSet):
     # Handles all CRUD operations.
     queryset = list_haso_applications()
     serializer_class = HasoSerializer
+    permission_classes = [IsAuthenticatedOrReadOnly]
 
 
 class HitasApplicationViewSet(viewsets.ModelViewSet):
     # Handles all CRUD operations.
     queryset = list_hitas_applications()
     serializer_class = HitasSerializer
+    permission_classes = [IsAuthenticatedOrReadOnly]
```

Comment:
Was this for test purposes? If so, recommend that we don't leave tests to the code that we don't forget to remove those later.